app/src/main/kotlin/com/bitwarden/authenticator/ui/platform/feature/settings/SettingsScreen.kt

```diff
@@ -294,6 +322,25 @@ private fun VaultSettings(
         dialogConfirmButtonText = stringResource(R.string.learn_more),
         dialogDismissButtonText = stringResource(R.string.ok),
     )
+    if (shouldShowSyncWithBitwardenApp) {
```

Comment:
Can we get a test to validate this is shown/hidden based on state?

Reply:
Added :)